freemocap/fmc_good_frame_finder.py

```diff
@@ -0,0 +1,103 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+
+def find_velocity_values_within_limit(skeleton_velocity_data, velocity_limit):
+    """
+    This function takes in a skeleton velocity data array and a limit and returns the indices of the values that are within the limit
+    """
+    indices = []
+    for i in range(len(skeleton_velocity_data)):
+        if abs(skeleton_velocity_data[i]) <= velocity_limit:
+            indices.append(i+1) #add 1 to account for the difference in indices between the position and velocity data
+    return indices
+
+def find_matching_indices_in_lists(list_1,list_2,list_3,list_4):
```

Comment:
I think this is probably going to run pretty slow.  again good enough for now, but we can chat about it for future iterations